tests/SBRPTests/SBRPTests.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace SBRPTests;
+
+public class ReferencePackagesTest : IClassFixture<SBRPFixture>, IDisposable
+{
+
+    public string WorkingDirectory { get; set; }
+    public ReferencePackagesTest(SBRPFixture fixture)
+    {
+        WorkingDirectory = fixture.WorkingDirectory;
+    }
+
+    [Theory(Timeout = 300000)]    
```

Comment:
This seems like a really long timeout.  What is the default?  Have you encountered this being an issue?

Reply:
Timout feature seems not work. I tried to set it to 1 and the tests still could finish correctly. So just removed the timeout.